### PR TITLE
Fix bad ens:// protocol (E_MISSING_ONE_SLASH)

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Review by mikehelmick — APPROVED** [review]

File: pkg/controller/redirect/index.go [hunk 1]
```diff
@@ -34,8 +35,15 @@ func (c *Controller) HandleIndex() http.Handler {
 
 		for hostname, region := range c.hostnameToRegion {
 			if host == hostname {
-				sendTo := fmt.Sprintf("ens:/%s&r=%s", path, region)
```
Comment (flagxor) [inline pkg/controller/redirect/index.go:37]: FWIW, this was "okay" in the sense that path has a leading slash.
But building the URI explicitly is better.

**Review by flagxor — APPROVED** [review]
lgtm

**Review by sethvargo — COMMENTED** [review]
Updated and added some tests 

/unhold
/assign @mikehelmick

**Review by mikehelmick — APPROVED** [review]

**Comment by flagxor** [comment]
Actually this introduced a regression (it eats the codes now).
Fixing as part of the url CL.

**Pull request merged**